docs/source/bond_family_guide/users_guide/tutorial.rst

```diff
@@ -0,0 +1,524 @@
+
+********
+Tutorial
+********
+
+Overview
+========
+
+This tutorial is a guide to the use of the Sawtooth Lake Bond Family user
+interface (UI).
+
+Before You Begin
+================
+
+Please :doc:`contact <../sysadmin_guide/contacts>` your Intel representative to
+receive the URL that you will connect to, and other needed information.
+
+Considerations
+==============
+
+The Sawtooth Bond Family UI runs in a cloud-based instance that is installed,
+configured and maintained by Intel. You do not need to install any specialized
+sofware on your  computer.
+
+Supported Browsers
+==================
+
+The following web browsers are supported:
+
+#. Firefox version 47.0+
+#. Chrome version 52.0+
+#. Safari version 9.1+
+#. Internet Explorer version 11+
+
+Supported Tasks
+===============
+
+You can perform the following tasks in the Sawtooth Bond Family UI:
+
+Account Creation and Updates
+----------------------------
+
+- `Create an Account`_
+- `Sign in with Existing Account`_
+- `Update User Account`_
+
+
+Buying and Selling 
+------------------
+
+- `View Bonds`_
+- `View Quotes`_
+- `Issue a Quote`_
+- `Create an Order`_
+- `Settle a Matched Order`_
+
+
+Holdings and Transactions
+-------------------------
+
+- `View Portfolio`_
+
+
+Administrative 
+--------------
+
+- `Create Organization`_
+- `Create Bond`_
+
+
+Tasks Not Supported by the UI
+=============================
+
+Certain tasks are not supported by the UI,  
+but can be performed using the command-line interface (CLI).
+For example, bond redemption can be accomplished with the CLI.
+
+See the :doc:`CLI reference guide <cli>` for information on performing tasks
+not supported by the UI, but which are supported by the CLI.
+
+Create an Account
+===================
+
+The first time you sign in to the Sawtooth Lake Bond UI, you need to
+create your account. During this process, you generate a new wallet import
+format (WIF) key, or import an existing WIF key. Your key is used to identify
+participants and sign transactions.
+
+.. note:: You will remain signed in to the Bond UI as long as your browser 
+          stores the cookies used by the Bond UI web server to identify 
+          you. If you delete the cookies, or need to sign in from 
+          another browser, simply follow the procedure below entitled
+          `Sign in with Existing Account`_. 
+
+
+Creating an Account with a New Key
+----------------------------------
+
+To create an account with a new WIF key:
+
+#. Navigate to the URL of the Bond UI instance.
+
+   - The URL is provided to you by :doc:`your Intel representative <../sysadmin_guide/contacts>`. 
+
+#. Click **Generate WIF**.
+
+#. Save your WIF key using one of the following methods:
+
+   - Click **Download Key** to download the WIF key to your local filesystem.
+   - Alternatively, copy the WIF key to your clipboard by clicking **Copy Key
+     to Clipboard**.
+
+      .. note:: If you use the Safari web browswer, you must press "Command-c" 
```

Comment:
This is no longer true. Safari has since been updated, and now supports copying fine. Although it does _not_ support downloading.